src/js/common/components/DataTable/DataTableHead.jsx

```diff
@@ -25,13 +26,15 @@ const DataTableHead = ({
   return (
     <TableHead className={className}>
       <TableRow>
-        <TableCell>
-          <Checkbox
-            color='primary'
-            onChange={onSelectAllClick}
-            checked={rowCount > 0 && numSelected === rowCount}
-            indeterminate={numSelected > 0 && numSelected < rowCount}
-          />
+        <TableCell padding='checkbox'>
+          {disableCheckbox ? undefined : (
```

Comment:
I think that you can use the prop `disableCheckbox` to hide the `<TableCell>` on line 29 too. You can use the `startExtraCells` prop to add an extra `<TableCell>` in this component: `src/js/views/createDevice/steps/SecurityStep/SecurityTable.jsx`.